examples/example-expo/e2e/mock-server.ts

```diff
@@ -20,7 +19,7 @@ export const createMockServer = (): [any, jest.Mock<MockRequest, any>] => {
   const handleRequest = (req: Request, res: Response) => {
     let body
     try {
-      body = req.body.compression === 'lz64' ? JSON.parse(LZString.decompressFromBase64(req.body.data) || '') : req.body
+      body = req.body
```

Comment:
Can this really error?  I wouldnt think so

Reply:
removed try catch cus tests were green anyway, we were not asserting anything with the log error call